platform/linux-generic/include/odp/api/plat/packet_inline_types.h

```diff
@@ -115,7 +115,7 @@ typedef union {
 	uint32_t all_flags;
```

Comment:
Commit subject and description: use singular API, avoid plural. Actually mentioning API at all may be redundant, since practically any implementation implements some part of the API.

Reply:
Sure, dropped the plural. Keeping the API word still as this is a multi-commit feature and in this commit just the actual interface level functions are modified (with one big squashed commit it would be a bit pointless yes).